lib/python/vdm/tests/server/start_stop_database_test.py

```diff
@@ -0,0 +1,150 @@
+"""
+This file is part of VoltDB.
+
+Copyright (C) 2008-2016 VoltDB Inc.
+
+This file contains original code and/or modifications of original code.
+Any modifications made by VoltDB Inc. are licensed under the following
+terms and conditions:
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+"""
+
+import unittest
+import requests
+import xmlrunner
+import socket
+import time
+
+__host_name__ = socket.gethostname()
+__host_or_ip__ = socket.gethostbyname(__host_name__)
+
+URL = 'http://%s:8000/api/1.0/databases/1/servers/' % \
+(__host_or_ip__)
+__db_url__ = 'http://%s:8000/api/1.0/databases/' % \
+             (__host_or_ip__)
+
+
+class Server(unittest.TestCase):
```

Comment:
Could you please add a test that uses different ports other than the default ports and tests that start and stop will work correctly?